test/support/environment-variables.ts

```diff
@@ -17,6 +19,8 @@ export const ENVIRONMENT_VARIABLES = Object.freeze({
   ACBS_AUTHENTICATION_PASSWORD: valueGenerator.string(),
   ACBS_MAX_REDIRECTS: 0,
   ACBS_TIMEOUT: 1000,
+  API_KEY: process.env.API_KEY,
+  API_KEY_STRATEGY: process.env.API_KEY_STRATEGY,
```

Comment:
Instead of loading this from the `.env` file, can we set them to a constant for the tests (either using `valueGenerator` or not)? That way it doesn't rely on the developer setting them in the `.env` file